File: trim_sales.py
with open('src/pages/SalesPage.jsx', 'r', encoding='utf-8') as f:
    lines = f.readlines()
with open('src/pages/SalesPage.jsx', 'w', encoding='utf-8') as f:
    f.writelines(lines[:440])
print('Done. Kept', len(lines[:440]), 'lines')

File: trim_sales.mjs
import { readFileSync, writeFileSync } from 'fs';
const content = readFileSync('src/pages/SalesPage.jsx', 'utf8');
const lines = content.split('\n');
writeFileSync('src/pages/SalesPage.jsx', lines.slice(0, 440).join('\n'), 'utf8');
console.log('Done, kept', Math.min(440, lines.length), 'lines, total was', lines.length);
